components/datadog/updater/install_script.sh

```diff
@@ -98,5 +94,18 @@ elif [ "${OS}" = "RedHat" ]; then
     $sudo_cmd sh -c "echo -e '[datadog]\nname = Datadog, Inc.\nbaseurl = https://${yum_url}/${yum_repo_version}/${ARCH}/\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\npriority=1\ngpgkey=${gpgkeys}' > /etc/yum.repos.d/datadog.repo"
     $sudo_cmd yum -y clean metadata
     $sudo_cmd yum -y install datadog-installer
+elif [ "${OS}" = "SUSE" ]; then
+    yum_url="yumtesting.datad0g.com/suse/testing"
+    yum_url="yumtesting.datad0g.com/testing"
```

Comment:
```suggestion
```
Otherwise you override the previous line with something that won't work on suse no?

Reply:
Indeed, however it's interesting that I was able to create a VM with that code :thinking: 

Maybe there aren't that many differences and we could just merge SUSE & RPM?!